bundles/org.openhab.binding.tado/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -200,4 +204,27 @@
 		<description>ON if at home, OFF if away</description>
 	</channel-type>
 
+	<channel-type id="batteryLowAlarm">
```

Comment:
Please don't introduce your own channel type for this but use `system.low-battery` instead (see https://www.openhab.org/docs/developer/bindings/thing-xml.html#system-state-channel-types).

Reply:
Oops, just saw that this had been discussed in https://github.com/openhab/openhab2-addons/pull/6057#discussion_r324465798 already.
If the system channel type is NOT read only, I would consider this a bug. I hence would still ask you to make use of it. (and the description should be fine, because if it is a channel of the zone, it is somewhat clear that the lowBattery is related to the zone).

Reply:
Ok Kai, I will revert to the system defined channel type. But can you please — therefore —raise a bug report that the system defined channel type must have the read only attribute tag?

Reply:
PS @kaikreuzer this binding also has temperature and humidity channels in the original code. So I suppose that if you want to force the usage of the system defined channel types, then logically I should also do that for those channels too. Or?? Personally I was trying to minimise my intervention in the original code to avoid breaking anything, but I can make those changes also, if you wish me too. What do you prefer? Is there a convention in OH about other persons making retrospective changes to other persons code?

PPS I suppose that I should therefore also go back to my other two bindings [siemensrds] and [neohub] and modify them to use system defined channel types for humidity and temperature? Or??

Reply:
> raise a bug report that the system defined channel type must have the read only attribute tag?

The system channel is defined as readOnly=true, see [here](https://github.com/openhab/openhab-core/blob/0f74348f5111fe94ef8069078bf57a426e1bca1b/bundles/org.openhab.core.thing/src/main/java/org/eclipse/smarthome/core/thing/DefaultSystemChannelTypeProvider.java#L84). Can you really confirm that this is not the case? If so, feel free to enter a bug report, stating how you notice that this is not working as expected.

> So I suppose that if you want to force the usage of the system defined channel types, then logically I should also do that for those channels too. Or??

We do not ask anybody to adapt existing channels (as this might have some small impact on existing setups (e.g. label texts may change), but for newly introduced channels, we definitely wish that the system channel types are used.

Reply:
Ok. Thanks